docs/conf.py

```diff
@@ -31,14 +31,18 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('../'))  # NOQA
+
 from anitya import app
 
 
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.doctest',
     'sphinx.ext.viewcode',
-    'sphinx.ext.githubpages',
```

Comment:
I recommend making this change in a separate commit since it's a separate concern.